jdk/test/java/awt/datatransfer/HTMLDataFlavors/HTMLDataFlavorTest.java

```diff
@@ -21,15 +21,19 @@
  * questions.
  */
 
-/*
-  @test
-  @bug 7075105
-  @summary WIN: Provide a way to format HTML on drop
-  @author Denis Fokin: area=datatransfer
-  @build HtmlTransferable PutAllHtmlFlavorsOnClipboard
-  @build PutOnlyAllHtmlFlavorOnClipboard PutSelectionAndFragmentHtmlFlavorsOnClipboard
-  @run main HTMLDataFlavorTest
-*/
+/**
+ * @test
+ * @key headful
+ * @bug 7075105
+ * @summary WIN: Provide a way to format HTML on drop
+ * @author Denis Fokin: area=datatransfer
+ * @requires (os.family == "windows")
+ * @library ../../../../lib/testlibrary
+ * @build HtmlTransferable PutAllHtmlFlavorsOnClipboard
+ * @build PutOnlyAllHtmlFlavorOnClipboard PutSelectionAndFragmentHtmlFlavorsOnClipboard
+ * @build jdk.testlibrary.OSInfo
```

Comment:
will this new library works in jdk8? did you test it?

Reply:
I tested it locally, and the new library is working.

Reply:
But why did you add it? In jdk10 it was part of the header before the patch, but in jdk8 we do not have that.

Reply:
deleted "@library ../../../../lib/testlibrary" and "@build jdk.testlibrary.OSInfo".